backend/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -46,6 +48,10 @@ public EmailTemplate get(String bundle, String application) {
             if (application.equalsIgnoreCase(APP_ANSIBLE_REPORTS)) {
                 return new Ansible();
             }
+        } else if (bundle.equalsIgnoreCase(PLATFORM)) {
+            if (application.equalsIgnoreCase(NOTIFICATIONS)) {
+                return new PlatformNotifications();
```

Comment:
Should this be renamed to ConsoleNotifications?

Reply:
I guess so. Probably does not matter atm.